examples/upbound-aws-provider/composite-resources/databases/dynamo-irsa/claim/dynamo-irsa.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: awsblueprints.io/v1alpha1
+kind: DynamoIRSA
+metadata:
+  name: dynamo-irsa-test
+spec:
+  writeConnectionSecretToRef:
```

Comment:
Use `publishConnectionDetailsTo` instead.

Reply:
I tried had issues with it. Atm all the compositions in the upbound-aws-provider section use `writeConnectionSecretToRef`. I will change at a later time across the board.

Reply:
What issue did you have? It does work and we really should be using it for new compositions.